src/Microsoft.AspNetCore.OData/ODataController.cs

```diff
@@ -51,5 +51,62 @@ protected virtual UpdatedODataResult<TEntity> Updated<TEntity>(TEntity entity)
 
             return new UpdatedODataResult<TEntity>(entity);
         }
+
+        /// <summary>
+        /// Creates a <see cref="StatusCodeResult"/> that when executed will produce a Bad Request (400) response.
+        /// </summary>
+        /// <returns>A <see cref="BadRequestResult"/> with the specified values.</returns>
+        public override BadRequestResult BadRequest()
+        {
+            return new BadRequestODataResult("Bad request");
```

Comment:
>"Bad request" [](start = 45, length = 13)

why hard code the string here? If it's hard code, it should be in to the "BadRequestODataResult"

Reply:
Removed the entire method.
#Resolved